core/src/impl/Kokkos_Core.cpp

```diff
@@ -750,12 +750,14 @@ void pre_initialize_internal(const Kokkos::InitializationSettings& settings) {
 #elif defined(KOKKOS_ARCH_BLACKWELL120)
   declare_configuration_metadata("architecture", "GPU architecture",
                                  "BLACKWELL120");
+// clang-format off
 #elif defined(KOKKOS_ARCH_AMD_GFX906)
       declare_configuration_metadata("architecture", "GPU architecture",
                                      "AMD_GFX906");
 #elif defined(KOKKOS_ARCH_AMD_GFX908)
       declare_configuration_metadata("architecture", "GPU architecture",
                                      "AMD_GFX908");
+// clang-format on
```

Comment:
I'd prefer the more consistent indentation
```suggestion
// clang-format off
#elif defined(KOKKOS_ARCH_AMD_GFX906)
  declare_configuration_metadata("architecture", "GPU architecture",
                                 "AMD_GFX906");
#elif defined(KOKKOS_ARCH_AMD_GFX908)
  declare_configuration_metadata("architecture", "GPU architecture",
                                 "AMD_GFX908");
// clang-format on
```